src/Public/Routes.ps1

```diff
@@ -430,6 +430,8 @@ function Add-PodeRoute {
                     Arguments        = $ArgumentList
                     Method           = $_method
                     Path             = $Path
+                    IsAsync          = $false
+                    AsyncPoolName    = "__$($_method)$($Path)_$($_endpoint.Name)_".Replace('/', '_')
```

Comment:
I made this change because the ID looked cluttered with too many //// in the RESTful call. I'm also concerned about the potential complications with all the necessary escaping when writing your client

Reply:
I changed the name to AsyncRouteId